.github/workflows/jobs-integration-test.yml

```diff
@@ -0,0 +1,59 @@
+name: Integration test
+on:
+  workflow_call:
+    inputs:
+      environment_name:
+        type: string
+        required: true
+        description: The name of the environment.
+      project_prefix_name:
+        type: string
+        required: true
+        description: The prefix name of the project
+    secrets:
+      AZURE_CREDENTIALS:
+        required: true
+    
+env:
+  Bot1Url: "https://byk-dev-aks-ingress.westeurope.cloudapp.azure.com/mock-bot"
```

Comment:
I think it would be better to move all bar the `keyvault_name` variable down to the `Test Test repo` step so all the test configuration values are together.

Reply:
It is actually used in the "Get secrets from key vault" step but I'll move it there

Reply:
Have moved all env vars to the step where they are used